### PR TITLE
#279 changed developer tools to development tools

### PR DESCRIPTION
**Pull request by digitaldiina**

### CODE REVIEW

File: docs/ui/soprotocol/index.md [hunk 1]
```diff
@@ -25,10 +25,7 @@ For example, to go to the contact with contact_id=2:
 ![08][img4]
 
 > [!TIP]
-> You can get a URL that reflects the current state of the application by selecting **Copy shortcut** either from the hamburger menu, the **Task** menu in dialogs for follow-ups and documents, or by right-clicking an item in a section tab,
-
-> [!TIP]
-> Can get from address bar, and from Developer Tools: (SuperOffice.Environment.SoProtocol)
+> You can get a URL that reflects the current state of the application by selecting **Copy shortcut** either from the hamburger menu, the **Task** menu in dialogs for follow-ups and documents, or by right-clicking an item in a section tab, can get from address bar, and from Development Tools: (SuperOffice.Environment.SoProtocol)
```
Comment (SuperOfficeDevNet) [inline docs/ui/soprotocol/index.md:28]: This one is NOT specific to the license. It refers to the Browser Developer Tools, so this one is OK.
image

**Review by SuperOfficeDevNet — COMMENTED** [review]
Looks good. Just need to adjust the last file, so protocol is referring to browser developer tools, not SuperOffice license information.

File: docs/ui/soprotocol/index.md (hunk 1 above)
Comment (digitaldiina) [reply docs/ui/soprotocol/index.md]: updated

**Review by SuperOfficeDevNet — APPROVED** [review]
Thanks.

**Pull request merged**